components/ProposalCard.tsx

```diff
@@ -87,15 +87,13 @@ export default function ProposalCard({
   status,
   expires_at,
 }: ProposalCardProps) {
-  const expiresAtDateTime = new Date(expires_at / 1000000).toLocaleString()
+  //// TODO fix, expires_at time
```

Comment:
I use this convention for todos: `TODO(user.name): message` which is nice for grepping with a lot of contributors.

Reply:
Added a link to the new git issue I just created for this.